### ISSUE TITLE
Failed to repeat the experiment CPGA published results with your code in master branch.

### ISSUE THREAD

**Issue by Bostoncake** [SCUT-AILab/CPGA#1]
Hello!
I tried to repeat the results by running the code according to the directions in README.md. Here are some issues I encountered in the process:

- It seems that the code could not normally run without hotfixes, since some args you used in the code were not defined when running the python scripts you mentioned.
- When training the target module on validation set, NaN warning is spectated during training.
- After 40/400 epochs, I couldn't repeat the results since the best module saved only reach 40%-60% accuracy. 

Would you please ensure that the code in your master branch IS the code you used for your publication? If so, could you please provide a more specific training instruction, or a specific training recording reaching the results you mentioned in your publication? 

Thank you!

**Comment by zxr177** [comment]
Hello, I am reproducing this project recently. May I ask if you have reproduced successfully? I encountered some minor problems when reproducing

**Comment by Bostoncake** [comment]
> Hello, I am reproducing this project recently. May I ask if you have reproduced successfully? I encountered some minor problems when reproducing

I couldn't reproduce the reported results. The best I could get was 62.5~ as for VisDA-C dataset. NaN appears most of the time when running training scripts.

**Comment by zxr177** [comment]
Thank you very much for your reply. I encountered a code problem while reproducing, and I don’t know how to solve it. If it is convenient for you, can you contact me at this mailbox 2415169235@qq.com to ask about the code error, very grateful.

**Comment by qiuzhen8484** [comment]
> Hello! I tried to repeat the results by running the code according to the directions in README.md. Here are some issues I encountered in the process:
> 
> * It seems that the code could not normally run without hotfixes, since some args you used in the code were not defined when running the python scripts you mentioned.
> * When training the target module on validation set, NaN warning is spectated during training.
> * After 40/400 epochs, I couldn't repeat the results since the best module saved only reach 40%-60% accuracy.
> 
> Would you please ensure that the code in your master branch IS the code you used for your publication? If so, could you please provide a more specific training instruction, or a specific training recording reaching the results you mentioned in your publication?
> 
> Thank you!

Since the number of training epochs of the generator was set to 0 in default,  you may get poor performance if you do not notice this.
We have solved the mentioned problems. 
Thanks for your attention.
